README.md

```diff
@@ -62,12 +62,23 @@ The Dockerfile and docker-compose.yml files replicate the steps detailed below f
 
 Once the Docker Compose server is running, the logs should provide the ngrok URL. For example: `SIMPLE_SERVER_HOST=91a705dde8c1.ngrok.io`. This is the value that should be used when setting up the Android app as described in the section below.
 
+After a successful docker installation, an admin dashboard account is automatically created.
+
+The default username...
```

Comment:
Oops, I just used `...` to refer to the next sentence. We can make this `The default username and password for the dashboard account is:`